app/models/post.rb

```diff
@@ -0,0 +1,4 @@
+class Post < ApplicationRecord
+  has_many :comments,dependent: :destroy
+  has_many :likes,dependent: :destroy
```

Comment:
Space missing after comma
```suggestion
  has_many :comments, dependent: :destroy
  has_many :likes, dependent: :destroy
```